### ISSUE TITLE
Can't compile on Ubuntu 13.04 

### ISSUE THREAD

**Issue by allan-simon** [mildsunrise/ccv15#3]
There's plenty of windows-only headers:

```
#include <windows.h>
#include <conio.h>
```

etc.

As I've already seen a binary for Linux, I was wondering how it was compiled in a first place?

**Comment by mildsunrise** [comment]
This repository contains the modified source code that was used to build the binary. You can browse the [commits](https://github.com/jmendeth/ccv15/commits/master) to look at each change I made after cloning the original source from SVN.

Now on how to build CCV. It's really confusing, and we definitively need clear instructions on how to do so. This is _not my fault_, since I just grabbed the source and made it compatible.

Look at the [`readme.linux`](https://github.com/jmendeth/ccv15/blob/master/readme.linux) file for intructions. There are scripts that automate the process of installing CodeBlocks (the IDE used to build CCV) and the dependencies needed to build CCV.

I haven't tried the scripts, and they're old so they might no longer work.

**Comment by allan-simon** [comment]
@jmendeth  thanks for your answer,  actually I've followed the instructions, but what is weird is that the compiliation failed with codeblock, but it's actually nothing due to "outdated" library or so as it's because he can't find  windows.h  (which is a Microsoft Windows include files, and the code is full of  DWORD, GUID  etc.  defined by this include )

I've edited my first message as due to markdown, the include where not correctly rendered in github

**Comment by mildsunrise** [comment]
`Windows.h`?! :S
I thought I had removed/conditioned all the occurences of `windows.h`
in CCV's source. I'll look it up.

**Comment by mildsunrise** [comment]
@allan-simon Protip: indent blocks of code by four spaces, it'll be treated as it deserves. [Markdown is easy.](http://stackoverflow.com/editing-help)

**Comment by allan-simon** [comment]
@jmendeth  thanks, if you need any help so that we can split the tasks, just tell me. I really appreciate the work you've already made and I would like to contribute back :)

**Comment by mildsunrise** [comment]
So I just cloned the repo, ran

``` bash
$ sudo apt-get install codeblocks
```

followed by

``` bash
$ sudo scripts/linux/ubuntu/install_dependencies.sh
```

Then I opened the project in Code::Blocks (`apps/addonsExamples/Codeblocks_32bit/Community Core Vision.cbp`) and started building. I didn't find any problem regarding `windows.h`.

**Note:** you'll notice there's a `Codeblocks_64bit` folder as well. I haven't touched that so it's plain wrong. Don't use it.

**Comment by mildsunrise** [comment]
Hey, I just copied the 32bit project to the 64bit folder (5f00a85e176ae610a2ddfe1f7c811d248b98054f) and adapted it to 64bit (fad63d6c64db431c463f2cdec7bdd452e545ffd6).
I tried it and it works as expected, but we need 64bit builds of the dependencies (see `libs/`).
I already did that for 32bit (see bea8b02b679b7c3e54173199f7f2c7430697368a, 1222f8e68180d7eaaf80d3bcb716e491b12d37e3 and ddd6df16844c35a0f969a97c280b17705f9e12b5) but 64bit is missing.
